tests/vic-machine.bats

```diff
@@ -125,6 +126,10 @@ teardown() {
     run "$docker" "${docker_cmd[@]}" start "$name"
     assert_success
 
+    wget "$vicadmin_ip"/container-logs.tar.gz --no-check-certificate -O install-container-logs.tar.gz
+    wget "$vicadmin_ip"/logs.tar.gz --no-check-certificate -O install-logs.tar.gz
+    assert_success
```

Comment:
You need to use 'run' to set '$status' before 'assert_success'.  And you need an 'assert_success' for each 'run wget ...'.

https://github.com/sstephenson/bats#run-test-other-commands

https://github.com/ztombol/bats-assert#assert_success